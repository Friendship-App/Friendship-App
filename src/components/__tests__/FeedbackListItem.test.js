import React from 'react';
import { shallow } from 'enzyme';
import FeedbackListItem from '../Feedback/FeedbackListItem';

describe('FeedbackListItem', () => {
  test('renders correctly', () => {
    const data = { title: 'This title' };
    const wrapper = shallow(<FeedbackListItem data={data} />);

    expect(wrapper).toMatchSnapshot();
  });
});
