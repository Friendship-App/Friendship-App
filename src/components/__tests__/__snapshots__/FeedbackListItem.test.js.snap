// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeedbackListItem feedbacklistitem recieve props correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FeedbackListItem
    data={
      Object {
        "title": "This title",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <View
          style={
            Object {
              "flex": 1,
              "flexWrap": "wrap",
              "marginLeft": 20,
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#4a4a4a",
                "fontFamily": "NunitoSans-Bold",
                "fontSize": 15,
              }
            }
          >
            This title
          </Text>
        </View>,
        <Icon
          allowFontScaling={false}
          name="ios-arrow-forward"
          size={12}
          style={
            Object {
              "fontSize": 20,
              "padding": 15,
              "textAlign": "center",
            }
          }
        />,
      ],
      "style": Object {
        "alignItems": "center",
        "borderBottomColor": "#d4d6d5",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingBottom": 7,
        "paddingTop": 7,
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#4a4a4a",
                  "fontFamily": "NunitoSans-Bold",
                  "fontSize": 15,
                }
              }
            >
              This title
            </Text>,
            undefined,
          ],
          "style": Object {
            "flex": 1,
            "flexWrap": "wrap",
            "marginLeft": 20,
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "accessible": true,
              "allowFontScaling": true,
              "children": "This title",
              "ellipsizeMode": "tail",
              "style": Object {
                "color": "#4a4a4a",
                "fontFamily": "NunitoSans-Bold",
                "fontSize": 15,
              },
            },
            "ref": null,
            "rendered": "This title",
            "type": [Function],
          },
          undefined,
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "allowFontScaling": false,
          "name": "ios-arrow-forward",
          "size": 12,
          "style": Object {
            "fontSize": 20,
            "padding": 15,
            "textAlign": "center",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <View
            style={
              Object {
                "flex": 1,
                "flexWrap": "wrap",
                "marginLeft": 20,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#4a4a4a",
                  "fontFamily": "NunitoSans-Bold",
                  "fontSize": 15,
                }
              }
            >
              This title
            </Text>
          </View>,
          <Icon
            allowFontScaling={false}
            name="ios-arrow-forward"
            size={12}
            style={
              Object {
                "fontSize": 20,
                "padding": 15,
                "textAlign": "center",
              }
            }
          />,
        ],
        "style": Object {
          "alignItems": "center",
          "borderBottomColor": "#d4d6d5",
          "borderBottomWidth": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "paddingBottom": 7,
          "paddingTop": 7,
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#4a4a4a",
                    "fontFamily": "NunitoSans-Bold",
                    "fontSize": 15,
                  }
                }
              >
                This title
              </Text>,
              undefined,
            ],
            "style": Object {
              "flex": 1,
              "flexWrap": "wrap",
              "marginLeft": 20,
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "accessible": true,
                "allowFontScaling": true,
                "children": "This title",
                "ellipsizeMode": "tail",
                "style": Object {
                  "color": "#4a4a4a",
                  "fontFamily": "NunitoSans-Bold",
                  "fontSize": 15,
                },
              },
              "ref": null,
              "rendered": "This title",
              "type": [Function],
            },
            undefined,
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "allowFontScaling": false,
            "name": "ios-arrow-forward",
            "size": 12,
            "style": Object {
              "fontSize": 20,
              "padding": 15,
              "textAlign": "center",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`FeedbackListItem renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FeedbackListItem
    data={
      Object {
        "title": "This title",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <View
          style={
            Object {
              "flex": 1,
              "flexWrap": "wrap",
              "marginLeft": 20,
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#4a4a4a",
                "fontFamily": "NunitoSans-Bold",
                "fontSize": 15,
              }
            }
          >
            This title
          </Text>
        </View>,
        <Icon
          allowFontScaling={false}
          name="ios-arrow-forward"
          size={12}
          style={
            Object {
              "fontSize": 20,
              "padding": 15,
              "textAlign": "center",
            }
          }
        />,
      ],
      "style": Object {
        "alignItems": "center",
        "borderBottomColor": "#d4d6d5",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingBottom": 7,
        "paddingTop": 7,
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#4a4a4a",
                  "fontFamily": "NunitoSans-Bold",
                  "fontSize": 15,
                }
              }
            >
              This title
            </Text>,
            undefined,
          ],
          "style": Object {
            "flex": 1,
            "flexWrap": "wrap",
            "marginLeft": 20,
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "accessible": true,
              "allowFontScaling": true,
              "children": "This title",
              "ellipsizeMode": "tail",
              "style": Object {
                "color": "#4a4a4a",
                "fontFamily": "NunitoSans-Bold",
                "fontSize": 15,
              },
            },
            "ref": null,
            "rendered": "This title",
            "type": [Function],
          },
          undefined,
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "allowFontScaling": false,
          "name": "ios-arrow-forward",
          "size": 12,
          "style": Object {
            "fontSize": 20,
            "padding": 15,
            "textAlign": "center",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <View
            style={
              Object {
                "flex": 1,
                "flexWrap": "wrap",
                "marginLeft": 20,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#4a4a4a",
                  "fontFamily": "NunitoSans-Bold",
                  "fontSize": 15,
                }
              }
            >
              This title
            </Text>
          </View>,
          <Icon
            allowFontScaling={false}
            name="ios-arrow-forward"
            size={12}
            style={
              Object {
                "fontSize": 20,
                "padding": 15,
                "textAlign": "center",
              }
            }
          />,
        ],
        "style": Object {
          "alignItems": "center",
          "borderBottomColor": "#d4d6d5",
          "borderBottomWidth": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "paddingBottom": 7,
          "paddingTop": 7,
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#4a4a4a",
                    "fontFamily": "NunitoSans-Bold",
                    "fontSize": 15,
                  }
                }
              >
                This title
              </Text>,
              undefined,
            ],
            "style": Object {
              "flex": 1,
              "flexWrap": "wrap",
              "marginLeft": 20,
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "accessible": true,
                "allowFontScaling": true,
                "children": "This title",
                "ellipsizeMode": "tail",
                "style": Object {
                  "color": "#4a4a4a",
                  "fontFamily": "NunitoSans-Bold",
                  "fontSize": 15,
                },
              },
              "ref": null,
              "rendered": "This title",
              "type": [Function],
            },
            undefined,
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "allowFontScaling": false,
            "name": "ios-arrow-forward",
            "size": 12,
            "style": Object {
              "fontSize": 20,
              "padding": 15,
              "textAlign": "center",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
